content/master/concepts/environment-configs.md

```diff
@@ -12,6 +12,20 @@ patches can read from and write to an environment.
 Crossplane supports multiple EnvironmentConfigs, each acting as a unique
 data store. 
 
+A Composition defines access to one or more EnvironmentConfigs.  
+
+When Crossplane creates a composite resource, Crossplane creates a unique copy
+of the defined EnvironmentConfigs for that composite resource.  
```

Comment:
```suggestion
When Crossplane reconciles a composite resource, Crossplane merges all the specified EnvironmentConfigs into an in-memory environment.
```

Reply:
Slight tweak, are you okay with this? 

> When Crossplane creates a composite resource, Crossplane merges all referenced
EnvironmentConfigs and creates a unique in-memory environment
for that composite resource.

Reply:
👍